modules/custom-status/lib/custom-status-block.js

```diff
@@ -78,7 +79,7 @@ const CustomSaveButtonSidebar = ( {
 
 	const buttonText = getCustomSaveButtonText( nextStatusTerm, isRestrictedStatus, isWideViewport );
 	const tooltipText = isRestrictedStatus
-		? __( 'Awaiting review from a privileged user', 'vip-workflow' )
+		? __( 'Requirements for advancing have not been met.', 'vip-workflow' )
```

Comment:
Would it be possible to have two messages here? One for when a status requires action (e.g. check a box), and one for when another user needs to make the change. I found myself looking through EM fields for a red asterisk whenever a status had a required user. This could be a frustrating experience for a user if it's unclear what is blocking movement when it's out of their control.

@jarekmorawski any thoughts or suggested copy here?

Reply:
This was part of what's not included in this PR yet - the UI inconsistencies, the red asterisk taking some time to show and missing backend checks for EM fields being required. 

Def agreed on this though that it needs to be improved further so it's not frustrating or unknown as to why it's blocked.